projects/dapr/cert-manager-graduation.md

```diff
@@ -0,0 +1,501 @@
+
+- [x] **All subprojects, if any, are listed.**
+
+Dapr treats each of its repositories [as a subproject](https://github.com/dapr/community/blob/master/community-membership.md).
+
+- [x] **If the project has subprojects: subproject leadership, contribution, maturity status documented, including add/remove process.**
+
+<!-- (TOC Evaluation goes here) --> 
+**TODO(dims)**: More work can be done here, dims to open a Issue
+
+## Contributors and Community
+
+Note: this section may be augmented by the completion of a Governance Review from TAG Contributor Strategy.
+
+### Suggested
+
+- [x] **Contributor ladder with multiple roles for contributors.**
+
+- [5 levels](https://github.com/dapr/community/blob/master/community-membership.md) of code related contributors and a [separate role for a community manager](https://github.com/dapr/community/blob/master/COMMUNITY-MANAGER.md) is defined in governance documentation.
+
+### Required
+
+- [x] **Clearly defined and discoverable process to submit issues or changes.**
+
+- [contributing-overview](https://docs.dapr.io/contributing/contributing-overview/) is defined in the docs website.
+
+- [x] **Project must have, and document, at least one public communications channel for users and/or contributors.**
+
+- [Discord](https://github.com/dapr/dapr?tab=readme-ov-file#community) is the platform of choice for online communications for all the participants.
+
+- [x] **List and document all project communication channels, including subprojects (mail list/slack/etc.).  List any non-public communications channels and what their special purpose is.**
+
+  In addition to the various channels on slack, `codeofconduct@dapr.io`, `stc@dapr.io` and `security@dapr.io` are some private mailing aliases to reach the respective folks.
+
+- [x] **Up-to-date public meeting schedulers and/or integration with CNCF calendar.**
+
+- [Bi-Monthly calls](https://tockify.com/cncf.public.events/monthly?search=dapr) are available in the CNCF calendar
+
+- [x] **Documentation of how to contribute, with increasing detail as the project matures.**
+
+- There are separate [Maintainer Guide](https://docs.dapr.io/contributing/docs-contrib/maintainer-guide/) and [Contributor Guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/) in addition to a lot of documentation in the [dapr/community](https://github.com/dapr/community) repository.
+
+- [x] **Demonstrate contributor activity and recruitment.**
+
+- Devstats has some good charts to show activity like the [users-statistics-by-repository-group](https://dapr.devstats.cncf.io/d/48/users-statistics-by-repository-group?orgId=1) and [companies-contributing-in-repository-groups](https://dapr.devstats.cncf.io/d/7/companies-contributing-in-repository-groups?orgId=1)
+- Github contributor graph shows [consistent activity](https://github.com/dapr/dapr/graphs/contributors) as well over time even as some people leave and others join.
+
+## Engineering Principles
```

Comment:
I've filed an issue to have this template header corrected: 
https://github.com/cncf/toc/issues/1455

Reply:
thanks @TheFoxAtWork